internal/dinosaur/pkg/services/dinosaur.go

```diff
@@ -328,12 +331,23 @@ func (k *dinosaurService) PrepareDinosaurRequest(dinosaurRequest *dbapi.CentralR
 		return nil
 	}
 
+	// Obtain organisation name from AMS to store in central request.
+	org, err := k.amsClient.GetOrganisationFromExternalID(dinosaurRequest.OrganisationID)
+	if err != nil {
+		return errors.OrganisationNotFound(dinosaurRequest.OrganisationID, err)
+	}
+	orgName := org.Name()
+	if orgName == "" {
+		return errors.OrganisationNameInvalid(dinosaurRequest.OrganisationID, org.Name())
```

Comment:
nitpick: We can use the variable here instead of calling the function again.
```suggestion
		return errors.OrganisationNameInvalid(dinosaurRequest.OrganisationID, orgName)
```